Rules/AvoidUnInitializedVarsInNewRunspaces.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
I haven't read the logic in detail yet but from a high level it looks ok, you are using the right methods and concepts to search through the AST, so feel free to continue